### PR TITLE
SpotifyPro-Integration: Add AudioAnalysis

### PR DESCRIPTION
**Pull request by Mattallmighty**
Audio analysis

### CODE REVIEW

File: src/pages/Settings/UICard.tsx [hunk 1]
```diff
@@ -174,7 +174,7 @@ const UICard = () => {
           />
         </div>
       )}
-      {showFeatures.integrations && (
```
Comment (YeonV) [inline src/pages/Settings/UICard.tsx:177]: @Mattallmighty not yet bro. I need to fix the "progressing seconds handling" first

File: src/store/ui/storeSpotifyActions.tsx [hunk 2]
```diff
@@ -169,10 +169,13 @@ const storeSpotifyActions = (set: any) => ({
...
         set(
           produce((state: any) => {
-            state.spotify.spTriggersList = [
-              ...state.spTriggersList,
-              newTrigger,
-            ];
+            // state.spotify.spTriggersList = [
+            //   ...state.spotify.spTriggersList,
+            //   newTrigger,
+            // ];
```
Comment (YeonV) [inline src/store/ui/storeSpotifyActions.tsx:175]: @YeonV test this

File: src/components/Dialogs/AddIntegrationDialog.tsx [hunk 3]
```diff
@@ -174,7 +174,9 @@ const AddIntegrationDialog = () => {
                 <MenuItem
                   key={i}
                   value={item}
-                  disabled={['mqtt_hass', 'spotify'].indexOf(item) === -1}
+                  disabled={
+                    ['mqtt_hass', 'spotify', 'qlc'].indexOf(item) === -1
+                  }
```
[the remark this answers is not in the document]
Comment (Mattallmighty) [reply src/components/Dialogs/AddIntegrationDialog.tsx:179]: Removing QLC to another Pull

File: src/components/AudioAnalysis/SpAnalysisLayoutBK.tsx [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+  albumURL: any;
+  name: any;
+  artist: any;
+  sections: any;
+  segments: any;
+};
+type MyState = { [key: string]: any };
+
+export class Layout extends React.Component<MyProps, MyState> {
+  constructor(props: any) {
```
[the remark this answers is not in the document]
Comment (Mattallmighty) [reply src/components/AudioAnalysis/SpAnalysisLayoutBK.tsx:29]: whoops haha, copy paste 
Yeah sure, will get this changed!

File: src/components/AudioAnalysis/SpAnalysisChordButtons.tsx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+              <Typography color="primary">{group}</Typography>
+            </InputLabel>
+            <Select
+              onChange={handleChordClick}
+              labelId={group}
+              color="primary"
+              className={classes.select}
+              inputProps={{ classes: { icon: classes.icon } }}
+            >
+              {Object.keys(chords[group]).map((chord) => {
```
[the remark this answers is not in the document]
Comment (Mattallmighty) [reply src/components/AudioAnalysis/SpAnalysisChordButtons.tsx:109]: done

**Pull request closed without merging**